urllib3/response.py

```diff
@@ -178,6 +188,8 @@ def read(self, amt=None, decode_content=None, cache_content=False):
             after having ``.read()`` the file object. (Overridden if ``amt`` is
             set.)
         """
+        if self.chunked:
+            return
```

Comment:
Does .read() simply stop working if there is a chunked header? Isn't that surprising?